src/extension/control-flow/src/main/java/org/geoserver/flow/config/DefaultControlFlowConfigurator.java

```diff
@@ -178,22 +185,75 @@ protected KeyGenerator buildKeyGenerator(String[] keys, String value) {
 
                     }.build(keys, value);
                 } else if (keys.length > 1) {
-                	if(!"blacklist".equals(keys[1]) && !"whitelist".equals(keys[1])){
-                		String ip = key.substring("ip.".length());
-                		controller = new SingleIpFlowController(queueSize, ip);
-                	}
+                    if (!"blacklist".equals(keys[1]) && !"whitelist".equals(keys[1])) {
+                        String ip = key.substring("ip.".length());
+                        controller = new SingleIpFlowController(queueSize, ip);
+                    }
                 }
             }
+            
             if (controller == null) {
                 LOGGER.severe("Could not parse rule '" + okey + "=" + value);
             } else {
+                LOGGER.info("Loaded flow controller configuration: " + key + "=" + value);
```

Comment:
This two logs messages are bit out of sync, I would have put something like:
```
LOGGER.severe("Could not parse control-flow rule '" + okey + "=" + value);
LOGGER.info("Loaded control-flow rule: " + key + "=" + value);
```

Reply:
Yup